queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT name, escape_attempts FROM animals WHERE animals.weight_kg > 10.5;
 SELECT * FROM animals WHERE animals.neutered = true;
 SELECT * FROM animals WHERE animals.name != 'Gabumon';
 SELECT * FROM animals WHERE animals.weight_kg >= 10.4 AND animals.weight_kg <= 17.3;
+
+START TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals WHERE animals.date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(escape_attempts) FROM animals WHERE animals.escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) as total_escape_attempts FROM animals GROUP BY neutered;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
```

Comment:
- [ ] [Optional] Nice job making the required changes, however, the query is supposed to get the average number of escape attempts per animal type of those born between the start of 1990 and the `end` of 2000, not the start of 2000 as you did. Kindly fix that.

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️